abstract/myst.yml

```diff
@@ -44,10 +35,10 @@ project:
 site:
   template: article-theme
   actions:
-    # - title: pdf
-    #   url: ARTICLE-NAME.pdf
-    #   static: true
-    # - title: slides
-    #   url: presentation/SLIDES-NAME.pdf
-    #   static: true
+    - title: poster
+      url: abstract/presentation/geophysics-in-a-changing-climate-eposter.pdf
+      static: true
```

Comment:
since this was a presentation with slides, we don't need to include a "poster", so this can be removed